controllers/drclusters.go

```diff
@@ -48,7 +48,7 @@ func drClusterDeploy(drClusterInstance *drclusterInstance, ramenConfig *rmn.Rame
 
 	annotations := make(map[string]string)
 
-	annotations["DRClusterName"] = mwu.InstName
+	annotations[DRClusterNameAnnotation] = mwu.InstName
```

Comment:
Is this leftover before we added `DRClusterNameAnnotation`?

Reply:
No, this is from the same [PR](https://github.com/RamenDR/ramen/pull/478/)

Reply:
OK, I guess it slipped in the review, this was a very big change.